packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -610,6 +633,15 @@ class RouteSettings {
 /// The type argument `T` is the corresponding [Route]'s return type, as
 /// used by [Route.currentResult], [Route.popped], and [Route.didPop].
 ///
+/// The [canPop] and [onPopInvoked] are used for intercepting pops.
```

Comment:
"The [canPop] and [onPopInvoked]" => "The [canPop] and [onPopInvoked] parameters"